### PR TITLE
fix(OnyxCheckbox,OnyxRadioButton,OnyxSwitch,OnyxSelect): reset control sizes to ignore density

### PR DESCRIPTION
**Pull request by BoppLi**
Relates to #1194

Reset control sizes to ignore density.
Fix RadioButton alignment to center with the label.

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/playwright/snapshots/components/OnyxRadioButton/Radio-button-truncation--chromium-linux.png
Comment (larsrickert) [inline packages/sit-onyx/playwright/snapshots/components/OnyxRadioButton/Radio-button-truncation--chromium-linux.png:1]: The radio button should be top aligned when multiline

File: packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue [hunk 1]
```diff
@@ -249,7 +233,7 @@ $input-width: calc(
       display: inline-flex;
       width: $input-width;
       min-width: $input-width;
-      height: var(--onyx-switch-input-height);
+      height: unset;
```
Comment (larsrickert) [inline packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.vue:236]: Why is this needed? Can't we just remove it?

**Review by larsrickert — CHANGES_REQUESTED** [review]
Still not a huge fan of the updates visuals because the densities can not really be differentiated any longer but ux decided like this so

File: packages/sit-onyx/playwright/snapshots/components/OnyxRadioButton/Radio-button-truncation--chromium-linux.png
Comment (larsrickert) [reply packages/sit-onyx/playwright/snapshots/components/OnyxRadioButton/Radio-button-truncation--chromium-linux.png:1]: Hmm but I remember that I discussed this with Jannick and he told me the default should be top aligned but the project can override it to be centered...lets double check this with @jannick-ux

Comment (BoppLi) [reply packages/sit-onyx/playwright/snapshots/components/OnyxRadioButton/Radio-button-truncation--chromium-linux.png:1]: ok I added it to the agenda of tomorrows discussion with jannick at 10:00

Comment (BoppLi) [reply packages/sit-onyx/playwright/snapshots/components/OnyxRadioButton/Radio-button-truncation--chromium-linux.png:1]: Answer from UX: top aligned is default

File: packages/sit-onyx/playwright/snapshots/components/OnyxCheckbox/Checkbox-other--chromium-linux.png
Comment (BoppLi) [inline packages/sit-onyx/playwright/snapshots/components/OnyxCheckbox/Checkbox-other--chromium-linux.png:1]: TODO optional got messed up

File: packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-hidden-label--chromium-linux.png
Comment (BoppLi) [inline packages/sit-onyx/playwright/snapshots/components/OnyxSwitch/Switch-hidden-label--chromium-linux.png:1]: todo this matrix does not make any sense anymore?

**Review by larsrickert — APPROVED** [review]

**Pull request merged**